common/src/main/java/io/netty/util/internal/ThreadLocalRandom.java

```diff
@@ -209,6 +225,12 @@ private static long newSeed() {
         }
     }
 
+    private static long mix64(long z) {
```

Comment:
@normanmaurer - o i c ... if the entire class is supposed to be a copy/paste then forget my comment :)